generator/v1api.go

```diff
@@ -173,6 +181,14 @@ var (
 		Name:        "stale",
 		In:          inQuery,
 	}
+	tokenAccessorParam = parameter{
+		Id:          "TokenAccessorParam",
```

Comment:
```suggestion
		Id:          "ACLTokenAccessorParam",
```

Same naming request.